commands/cdrom.go

```diff
@@ -214,6 +215,7 @@ func RunServerCdromDetach(c *core.CommandConfig) error {
 	if err := utils.AskForConfirm(c.Stdin, c.Printer, "detach cdrom from server"); err != nil {
 		return err
 	}
+	c.Printer.Infof("Cd rom with id: %v is detaching... ", viper.GetString(core.GetFlagName(c.NS, config.ArgCdromId)))
```

Comment:
Please update with CD-ROM or Cdrom, I think it would look better :)

Reply:
done, replaced with CD-ROM